evap/contributor/tests/test_views.py

```diff
@@ -122,6 +126,51 @@ def test_wrong_state(self):
         self.app.get(self.url, user="responsible@institution.example.com", status=403)
 
 
+# A submit function which conforms to the specification to not submit disabled fields.
+# See https://github.com/Pylons/webtest/issues/138
+def submit_fields_without_disabled(self, name=None, index=None, submit_value=None):
```

Comment:
did you write this yourself?

Reply:
No, I copied it from webtest’s sources `Form.submit_fields` and changed one line:

```patch
- if value is None:
+ if value is None or 'disabled' in field.attrs:
      continue
```

On second thought, this solution doesn’t really make anybody happy (codecov also flags some uncovered lines). During research, I stumpled across https://github.com/Pylons/webtest/issues/138#issuecomment-532760520 and would prefer this approach now – now we’re doing webtest’s job in the test, but this also gives us an opportunity to leave a comment about disabled fields :)

Reply:
just removing the `disabled` attribute sounds fine for me